Project.toml

```diff
@@ -6,6 +6,7 @@ version = "0.2.9"
 ColorVectorSpace = "c3611d14-8923-5661-9e6a-0046d554d3a4"
 Distances = "b4f34e82-e78d-54a5-968a-f98e89d6e8f7"
 ImageCore = "a09fc81d-aa75-5fe9-8630-4744c3626534"
+ImageMorphology = "787d08f9-d448-5407-9aad-5290dd7ab264"
```

Comment:
Also need to add a version compat here (every non-standard package in the dependency needs this so that we can automate the package registration process):

```yaml
ImageMorphology = "0.2"
```